docs/releases/25.8.1.md

```
@@ -0,0 +1,330 @@
+---
+title: Armbian 25.8.1
+description: "Armbian 25.8.1 release notes, published 26 August 2025: all 319 merged pull requests across boards, kernels, desktops and the build framework."
+---
+
+# Armbian 25.8.1
+
+*Released 26 August 2025*
+
+## All changes
+
+*  bump rockchip edge to kernel 6.16.  by @paolosabatino in [armbian/build#8435](https://github.com/armbian/build/pull/8435)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)